installer/terraform/chefProvisioner.tf

```diff
@@ -78,7 +96,7 @@ resource "null_resource" "configureExistingJenkinsServer" {
   depends_on = ["null_resource.updateJenkinsChefCookbook", "aws_s3_bucket.jazz-web", "null_resource.update_jenkins_configs"]
 
   connection {
-    host = "${lookup(var.jenkinsservermap, "jenkins_public_ip")}"
+    host = "${lookup(var.jenkinsservermap, "jenkins_elb")}"
```

Comment:
we might need jenkins_public_ip to ssh. Elb might have different IP each time.

Reply:
See prev comments.